src/module_tests.c

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#include "console.h"
+#include "pagetable.h"
+#include "process.h"
+
+void walk_memory() {
```

Comment:
Still seems like something useful to have available. Add it to the command line module?